### DISCUSSION TITLE
/etc/client_loop: send disconnect: Broken pipe after upgrage to 4.20

### DISCUSSION THREAD

**Discussion by Agvares666** [apache/cloudstack#10156] [Q&A] [answered]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VM
~~~

##### CLOUDSTACK VERSION

~~~
4.20
~~~

##### CONFIGURATION

basic network,  QEMU emulator version 4.2.1

##### OS / ENVIRONMENT

Ubuntu 20.04.6

##### SUMMARY

After upgrading from 4.19 to 4.20, all new VMs created from the template, after logging in to them via ssh, give /etc/client_loop: send disconnect: Broken pipe

##### STEPS TO REPRODUCE

~~~
Download the template, ssh to it, wait for 5 seconds
~~~

##### EXPECTED RESULTS

~~~
ssh session is not interrupted
~~~

##### ACTUAL RESULTS

~~~
ssh session is interrupted
~~~

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by DaanHoogland** [comment]
@Agvares666 , you talk about "the template". What template do you mean?

**Comment by weizhouapache** [comment]
it seems to be an issue with ssh configuration, not  cloudstack

**Comment by Agvares666** [comment, marked as answer]
Thank you all. It was solved by rebuilding the zone
